packages/ui/src/components/KafkaInstanceDrawer/components/SampleCodeSnippet.tsx

```diff
@@ -0,0 +1,81 @@
+import {
+  ClipboardCopyButton,
+  CodeBlock,
+  CodeBlockAction,
+  CodeBlockCode,
+  ExpandableSection,
+  ExpandableSectionToggle,
+} from "@patternfly/react-core";
+import { useState } from "react";
+import type { VoidFunctionComponent } from "react";
+
+export type SampleCodeSnippetProps = {
+  copyCode: string;
+  expandableCode?: string;
+  codeBlockCode: string;
+};
+
+export const SampleCodeSnippet: VoidFunctionComponent<
+  SampleCodeSnippetProps
+> = ({ copyCode, expandableCode, codeBlockCode }) => {
+  const [isExpanded, setIsExpanded] = useState(false);
+
+  const [copied, setCopied] = useState<boolean>(false);
+
+  const clipboardCopyFunc = (_event: any, text: string) => {
+    navigator.clipboard
+      .writeText(text.toString())
+      .then(() => {
+        setCopied(true);
+      })
+      .catch(() => {
+        setCopied(false);
+      });
+  };
+
+  const onToggle = (isExpanded: boolean) => {
+    setIsExpanded(isExpanded);
+  };
+
+  const actions = (
```

Comment:
Are these actions required? Isn't this the default behavior of the copy component from PF?

Reply:
No, By default actions will be null in code Block

Reply:
Bummer, ok